content/02.introduction.md

```diff
@@ -27,6 +28,10 @@ This is a real-time, collaborative effort that welcomes submissions from scienti
 #### Classification:
 Coronaviridae, commonly called Coronavirus, is a family of positive-sense RNA viruses [@isbn:0781760607]. 
 Members of this family include the Middle East respiratory syndrome-related coronavirus (MERS-CoV) and Severe acute respiratory syndrome coronavirus (SARS-CoV), which are two species, as well as SARS-CoV-2 [@url:https://en.wikipedia.org/wiki/Coronaviridae#Taxonomy].
+Coronaviridae are classified into four main groups called alphacoronaviruses, betacoronaviruses, gammacoronaviruses and deltacoronaviruses. 
+Alphacoronaviruses and betacoronaviruses are mainly found in mammals such as bats, rodents, civets, and humans, whereas gammacoronaviruses and deltacoronaviruses have their gene source from birds and are mainly found in birds.
```

Comment:
```suggestion
Alphacoronaviruses and betacoronaviruses are mainly found in mammals such as bats, rodents, civets, and humans, whereas gammacoronaviruses and deltacoronaviruses are mainly found in birds.
```

Reply:
I don't really know what "their gene source from birds" is supposed to mean but I would leave it out.

Reply:
Genetic source, like gene pool. I'm good to leave it out